src/receipt/_style.scss

```diff
@@ -0,0 +1,67 @@
+#receipt-container {
+    width: 1170px;
+    padding-top: 2rem;
+}
+
+.page__receipt {
+    h2 {
+        margin-bottom: 10px;
+        font-weight: 400;
+    }
+
+    .thank-you {
+        font-size: 38px;
+    }
+
+    .info-item {
+        overflow: hidden;
+    }
+
+    .copy {
+        > div {
+            margin-bottom: 1rem;
```

Comment:
Generally, it's preferred to use Paragon's SCSS spacer or utility classes than hard coded `rem` values for margins, e.g. `mb-3` or `map-get($spacers, 3)`.

Reply:
Not for this specifically, but sometimes I come across a case where I want a specific size that is not the same as bootstrap utility margin class spacer value, would I just revert to using a pixel or a fraction of a `rem` here?

Why is it preferred to use utility classes, because they render with the component as opposed to being compiled in a separate SCSS file? Just curious.

Reply:
@julianajlk 

> sometimes I come across a case where I want a specific size that is not the same as bootstrap utility margin class spacer value, would I just revert to using a pixel or a fraction of a rem here?

Personally, I find the closest existing spacer value to the pixel value I'm given. In an ideal world, the handoff from design -> engineering wouldn't refer to any hard coded pixel/rem values and would instead refer to an existing spacer value.

> Why is it preferred to use utility classes, because they render with the component as opposed to being compiled in a separate SCSS file? Just curious.
If spacing throughout the UI is using the same spacer values, it's trivial to update our spacing definitions in once place and have it apply everywhere. If we only use hard coded values for spacing, and our foundation spacing design tokens change (e.g., maybe `mb-3` is equivalent to `1rem` now, but `2rem` in the future), that change wouldn't propagate where spacing is hardcoded, leading to inconsistency in the UI.